website/docs/interactors/4-write-your-own.md

```diff
@@ -345,22 +353,24 @@ DatePicker().find(Button('31')).click();
 If the built-in DOM Interactors work for your use case, they are probably the best choice.
 They are maintenance-free and support the most common user actions.
 
-When the built-ins are not enough, it is normal and encouraged for you to write your own Interactors!
-They will help you prevent duplicated logic in your tests, and if your interface changes, you only need to make the change in one place.
+When the built-in Interactors are not enough, we encourage you to write your own. They will help prevent duplicated logic in your tests, and if your interface changes, you only need to make changes to the Interactor instead of throughout the code.
 
-For example, let's say that you want to replace a custom datepicker with a popular third-party library instead.
-Although you may have many tests for flows with the date picker, only your Interactor needs to change.
+For example, let's say that you want to replace a custom datepicker with a popular third-party library instead. Although you may have many tests for flows with the date picker, only your Interactor needs to change.
 
 ### I have an interaction that is really difficult to test. What should I do?
 
 A good test suite helps you uncover hidden problems.
 Often, difficult UI tests are your early warning system for areas of your app that may have accessibility issues.
```

Comment:
You can combine these two lines into one. Just a formatting thing. I think Jen likes to create new lines often when she works.